bindgen/dist/JSONBuilder.d.ts

```diff
@@ -1,6 +1,5 @@
 import { Node, FunctionDeclaration, Source, ClassDeclaration } from "visitor-as/as";
 import { BaseVisitor } from "visitor-as";
-export declare function toString(node: Node): string;
```

Comment:
In a follow-up PR, we should probably gitignore the `dist` directory; none of these artifacts need to be version controlled, and having them here just adds noise

Reply:
Some of our users depend on this repo directly (which they shouldn't) so keeping them won't break that.

Reply:
Then let's add `bindgen/dist` to `.gitattributes` so that GitHub knows to hide these diffs (we did this [in near-api-js](https://github.com/near/near-api-js/blob/master/.gitattributes))